src/main/java/com/reckue/post/services/realizations/CommentServiceRealization.java

```diff
@@ -31,25 +33,31 @@
 public class CommentServiceRealization implements CommentService {
 
     private final CommentRepository commentRepository;
-
     private final NodeRepository nodeRepository;
-
     private final PostRepository postRepository;
-
     private final NodeService nodeService;
+    private final TokenStore tokenStore;
 
     /**
      * This method is used to create an object of class Comment.
      *
      * @param comment object of class Comment
+     * @param token   user token
      * @return comment object of class Comment
      */
     @Override
     @Transactional
-    public Comment create(Comment comment) {
+    public Comment create(Comment comment, String token) {
```

Comment:
Need to create the service to work with tokens!

Reply:
We can do it in the nearest future.